tests/nn_test.py

```diff
@@ -370,5 +371,25 @@ def testVarianceScalingError(self):
       initializer(rng, shape)
 
 
+@jtu.with_config(jax_legacy_prng_key="allow", jax_numpy_dtype_promotion="standard")
+class InitializerDtypeTest(jtu.JaxTestCase):
```

Comment:
No need to create a new test suite just for this single test. Instead, you should adjust the flags locally as I mentioned in my previous comment.